src/controller/SetUpCodePairer.cpp

```diff
@@ -335,6 +370,36 @@ void SetUpCodePairer::OnBLEDiscoveryError(CHIP_ERROR err)
 }
 #endif // CONFIG_NETWORK_LAYER_BLE
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+void SetUpCodePairer::OnDiscoveredDeviceOverWifiPAF()
+{
+    ChipLogProgress(Controller, "Discovered device to be commissioned over WiFiPAF, RemoteId: %lu", mRemoteId);
+
+    mWaitingForDiscovery[kWiFiPAFTransport] = false;
+    auto param                              = SetUpCodePairerParameters();
+    param.SetPeerAddress(Transport::PeerAddress(Transport::Type::kWiFiPAF, mRemoteId));
+    mDiscoveredParameters.emplace_front(param);
```

Comment:
This should document why it's "jumping the queue" to go in the front.

Reply:
Yes, it's more reasonable to go to the end. I will change it.